configs/HGLRC_H743_PRO/config.h

```diff
@@ -0,0 +1,144 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU       STM32H743
+
+#define BOARD_NAME          HGLRC_H743_PRO
+#define MANUFACTURER_ID     HGLR
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_MAX7456
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
+#define SPI1_SCK_PIN            PA5
+#define SPI1_SDI_PIN            PA6
+#define SPI1_SDO_PIN            PA7
+#define GYRO_1_EXTI_PIN         PC5
+#define GYRO_1_CS_PIN           PA4
+#define GYRO_1_SPI_INSTANCE     SPI1
+#define USE_GYRO_CLKIN
+#define GYRO_1_CLKIN_PIN        PC6
+
+#define SPI2_SCK_PIN            PB13
+#define SPI2_SDI_PIN            PB14
+#define SPI2_SDO_PIN            PB15
+#define MAX7456_SPI_CS_PIN      PB12
+#define MAX7456_SPI_INSTANCE    SPI2
+
+#define SPI3_SCK_PIN            PB3
+#define SPI3_SDI_PIN            PB4
+#define SPI3_SDO_PIN            PB2
+#define FLASH_CS_PIN            PA15
+#define FLASH_SPI_INSTANCE      SPI3
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
```

Comment:
Flash uses SDMMC in the schematic. Check this and provided an updated schematic if necessary.